doc/source/lilac/obtaining-building-and-running/obtaining-and-building-ctsm.rst

```diff
@@ -231,7 +231,7 @@ Example usage for a Mac (a simple case) is::
 
   ./lilac/build_ctsm ~/ctsm_build_dir --os Darwin --compiler gnu --netcdf-path /usr/local --esmf-mkfile-path /Users/sacks/ESMF/esmf8.0.0/lib/libO/Darwin.gfortranclang.64.mpich3.default/esmf.mk --max-mpitasks-per-node 4 --no-pnetcdf
```

Comment:
this seems to be specific to @billsacks and not a general instruction

Reply:
@billsacks I'm happy to update this if want to send me the replacement.